README.md

```diff
@@ -45,6 +45,11 @@ $ gulp
 $ open http://localhost:8000/index.html
 ```
 
+Some components, like the [amp-user-notification(with backend service)](https://amp-by-example.appspot.com/amp-user-notification(with_backend_service).html) require a server to
+run.
+
+If you want to run the demo locally, you can use the server from [the amp-publisher-sample](https://github.com/ampproject/amp-publisher-sample) project.
+
```

Comment:
This won't work as the samples don't link to localhost. I don't think that we need this here.